### PR TITLE
add application resources and cluster status monitoring user guider

### PR DESCRIPTION
**Pull request by wanjunlei**
Signed-off-by: wanjunlei <wanjunlei@yunify.com>

### CODE REVIEW

File: content/en/docs/cluster-administration/cluster-status-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+|Monitoring indicators|Description|
+|---|---|
+|Request Latency|Classified statistics by HTTP request method, the latency of resource request response in milliseconds|
+|Request Per Second|The number of requests accepted by kube-apiserver per second|
+
+![APIServer Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/apiserver-monitoring.png)
+
+## Scheduler Monitoring
+
+[Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) Scheduler is the Kubernetes API of monitor the newly created Pod, and determine which nodes the newly created Pod should run on. It makes this decision based on available data, including the availability of collected resources and the resource requirements of the Pod. Monitoring data for scheduling delays ensure that you can see any delays faced by the scheduler.
```
Comment (Sherlock113) [inline content/en/docs/cluster-administration/cluster-status-monitoring.md:152]: ->
[Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) monitors the Kubernetes API of newly created pods and determines which nodes these new pods run on. It makes this decision based on available data, including the availability of collected resources and the resource requirements of the Pod. Monitoring data for scheduling delays ensures that you can see any delays facing the scheduler.

File: content/en/docs/cluster-administration/cluster-status-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+|---|---|
+|Attempt Frequency|Including the number of scheduling successes, errors, and failures|
+|Attempt Rate|Including scheduling rate of success, error, and failure|
+|Scheduler latency|End-to-end scheduling delay, which is the sum of scheduling algorithm delay and binding delay|
+
+![Scheduler Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/scheduler-monitoring.png)
+
+## Node Usage Ranking
+
+The node usage ranking is very useful for host monitoring. It supports perform ranking with indicators such as **CPU, Load Average, Memory, Local Storage, inode Utilization, and Pod Utilization**, support ascending and descending order. Administrators can quickly find potential problems or locate a node's insufficient resources by sorting by indicators.
```
Comment (Sherlock113) [inline content/en/docs/cluster-administration/cluster-status-monitoring.md:164]: It supports perform ranking with indicators such as **CPU, Load Average, Memory, Local Storage, inode Utilization, and Pod Utilization**, support ascending and descending order. Administrators can quickly find potential problems or locate a node's insufficient resources by sorting by indicators.
->
You can sort nodes in ascending and descending order by indicators such as **CPU, Load Average, Memory, Local Storage, inode Utilization, and Pod Utilization**. This enables administrators to quickly find potential problems or identify a node's insufficient resources.

**Review by Sherlock113 — COMMENTED** [review]
Please have a look at my comments and make necessary changes. Thanks.

I will make further changes to format and wording after this pr is merged.

**Comment by Sherlock113** [comment]
/lgtm
@FeynmanZhou Please merge this pr if you don't have other comments. I will make further format and wording changes in another pr. Thanks.

**Comment by FeynmanZhou** [comment]
> /lgtm
> @FeynmanZhou Please merge this pr if you don't have other comments. I will make further format and wording changes in another pr. Thanks.

ok

**Pull request merged**